packages/config/config/devices/0x027a/zse29.json

```diff
@@ -16,6 +16,244 @@
+					"label": "50",
+					"value": 50
+				},
+				{
+					"label": "100",
+					"value": 100
+				},
+				{
+					"label": "200",
+					"value": 200
+				},
+				{
+					"label": "300",
+					"value": 300
+				},
+				{
+					"label": "900 (Test mode)",
+					"value": 900
+				}
+			]
+		},
+		"5": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Motion Re-trigger Time",
+			"description": "Delay before the sensor reports no motion after detecting the last motion activity.",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 720,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Set by knob on sensor",
+					"value": 0
+				},
+				{
+					"label": "5 (Test mode)",
+					"value": 5
+				},
+				{
+					"label": "30",
+					"value": 30
+				},
+				{
+					"label": "60",
+					"value": 60
+				},
+				{
+					"label": "300",
+					"value": 300
+				},
+				{
+					"label": "600",
+					"value": 600
+				},
+				{
+					"label": "720",
+					"value": 720
+				}
+			]
+		},
+		"6": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Lux Reporting Frequency",
+			"unit": "minutes",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 1440,
+			"defaultValue": 30,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"7": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Manual Motion Re-trigger Time Setting",
+			"description": "Read the value for the Motion Re-trigger Time set by knob on sensor (overridden by setting Motion Re-trigger Time parameter).",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 5,
+			"maxValue": 720,
+			"defaultValue": 5,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "5 (Test mode)",
+					"value": 5
+				},
+				{
+					"label": "30",
+					"value": 30
+				},
+				{
+					"label": "60",
+					"value": 60
+				},
+				{
+					"label": "300",
+					"value": 300
+				},
+				{
+					"label": "600",
+					"value": 600
+				},
+				{
+					"label": "720",
+					"value": 720
+				}
+			]
+		},
+		"8": {
+			"$if": "firmwareVersion >= 2",
+			"label": "Manual Brightness Threshold Setting",
+			"description": "Read the value for the Brightness Threshold set by knob on sensor (overridden by setting Brightness Threshold parameter).",
+			"unit": "lux",
+			"valueSize": 2,
+			"minValue": 10,
+			"maxValue": 900,
+			"defaultValue": 900,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "10",
+					"value": 10
+				},
+				{
+					"label": "50",
+					"value": 50
+				},
+				{
+					"label": "100",
+					"value": 100
+				},
+				{
+					"label": "200",
+					"value": 200
+				},
+				{
+					"label": "300",
+					"value": 300
+				},
+				{
+					"label": "900 (Test mode)",
+					"value": 900
+				}
+			]
+		}
+	},
 	"metadata": {
```

Comment:
@AlCalzone With regard to the metadata, this is quoted directly out of the manual and is clearly not specific for the zwavejs2mqtt interface.  I see that the interface allows the user to display the metadata.  Should I edit it to refer to this app?  Or leave it alone?  (For example, instead of saying "Initiate inclusion (pairing) in the app (or web interface)..." it could say something like "Click Add/Remove Device ...")

Reply:
You can change this one I suppose, but we have around 2,400 files. That's why we just pull them from the manual.

Reply:
Happy to leave it as is; probably not a bad thing for user to see the direct quote from the manual. Wanted to make sure I wasn't leaving anything out.